internal/aws/aws.go

```diff
@@ -21,7 +22,7 @@ import (
 	"time"
 )
 
-func BucketRand(dryRun bool) {
+func BucketRand(dryRun bool, trackers map[string]*pkg.ActionTracker) {
 
```

Comment:
we could send the pointer of the tracker only, instead of the full `map`. As the function, doesn't need to be aware of all details.